ssl/ssl_locl.h

```diff
@@ -1349,6 +1349,19 @@ typedef struct sigalg_lookup_st {
     int curve;
 } SIGALG_LOOKUP;
 
+typedef struct {
+    int nid;                    /* Curve NID */
+    int secbits;                /* Bits of security (from SP800-57) */
+    uint16_t flags;             /* Flags: currently just group type */
+} TLS_GROUP_INFO;
+
+/* flags values */
+/* Mask for group type */
```

Comment:
this second comment confused me at first.  maybe indent or move it onto the define line?